slides/intro-04-evaluating-models.qmd

```diff
@@ -259,6 +259,70 @@ augment(forested_fit, new_data = forested_train) %>%
 countdown::countdown(minutes = 5, id = "roc-curve")
 ```
 
+## Brier score
+
+What if we don't turn predicted probabilties into class predictions?
+
+. . .
+
+The Brier score is analogous to the mean squared error in regression models:
+
+$$
+Brier_{class} = \frac{1}{N}\sum_{i=1}^N\sum_{k=1}^C (y_{ik} - \hat{p}_{ik})^2
+$$
+
+## Brier score
+
+```{r brier}
+augment(forested_fit, new_data = forested_train) %>% 
+  brier_class(truth = forested, .pred_Yes) 
+```
+
+. . .
+
+Smaller values are better, for binary classification the "bad model threshold" is about 0.25.
+
+
+## Separation vs calibration
+
+The ROC captures separation, the Brier score captures calibration.
```

Comment:
This is adapted from https://www.tidymodels.org/learn/statistics/survival-metrics/#disagreement-between-metrics

Reply:
This is a super artful slide.

I'm 50/50 on whether this is the right approach, but: the current structure of the slide _seems to suggest_ that the LHS plot refers to the comment on separation and RHS on calibration:

![Screenshot 2024-07-29 at 9 26 22 AM](https://github.com/user-attachments/assets/f5141caf-a5c6-4664-ba8f-29309cfdf9e8)

Should we leave no ambiguity / room for doubt there by including "The ROC captures separation" in the first `:::columns` and "The Brier score captures calibration" in the second?

Reply:
good point, I'll try it out!

Reply:
<img width="797" alt="Screenshot 2024-07-29 at 17 11 16" src="https://github.com/user-attachments/assets/b1588e51-65df-4c55-9469-e728732f5eca">